api/accounts/add.js

```diff
@@ -26,6 +27,13 @@ function addAccount (state, properties, options) {
 
   return state.db.put(doc)
 
+  .catch(function (error) {
```

Comment:
I might stick this `.catch` after the `.then`, but I believe they are functionally equivalent? Might just be a code style thing. @gr2m?